frontend/scss/components/organisms/component-sidebar.scss

```diff
@@ -8,6 +8,45 @@
 
 @import '_sidebar.scss';
 
+nav[toolbar] {
```

Comment:
Urgz. I guess this selector is required for `amp-sidebar`?

Reply:
Yeah. Truely sorry about that. :(